compliance/b_10.rst

```diff
@@ -0,0 +1,241 @@
+
+In such cases, Firely Server provides the flexibility to apply filtering parameters and/or by using groups, allowing for the export of specific subsets of patients.
+This accommodates both adherence to the requirement of exporting all patients and data, as well as the practical need for targeted data exports in specific situations.
+
+In the following examples, the first method exports the entire patient population, while the subsequent methods allows for exporting a subset of the patient population.
+
+
+Method 1: Exporting an Entire Patient Population using the Patient Endpoint
+---------------------------------------------------------------------------
+
+Perform a POST or GET request to the Patient endpoint without any filters.
+
+1. Create a POST or GET request to the Patient endpoint:
+
+        ``POST/GET {{BASE_URL}}/Patient/$export``
+
+This method exports all patients and their related data without applying any filtering parameters.
+
+Method 2: Exporting a Subset Patient Population using the Patient Endpoint
+--------------------------------------------------------------------------
+Perform a POST request to the Patient endpoint without any filters.
+
+1. Create a POST request to the Patient endpoint:
+
+        ``POST {{BASE_URL}}/Patient/$export``
+
+2. Include the example Parameters Resource from below in the request body.
+
+    .. code-block:: json
+
+        {
+            "resourceType": "Parameters",
+            "parameter": [
+            {
+                "name": "patient",
+                "valueReference": {
+                "reference": "Patient/test"
+                }
+            },
+            {
+                "name": "patient",
+                "valueReference": {
+                "reference": "Patient/other"
+                }
+            }
+            ]
+        }
+
+This method allows you to specify the patients to be exported by including their references in the Parameters Resource.
+
+Method 3: Exporting a Subset Patient Population using the Group Endpoint
+------------------------------------------------------------------------
+
+To export a subset of the patient population using the Group endpoint, you have two options:
+
+3.1. Exporting without filtering parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+        
+Perform a POST or GET request to the Group endpoint without any filters.
+
+1. Create a POST or GET request to the Group endpoint:
+
+        ``POST/GET {{BASE_URL}}/Group/[groupId]/$export``
+
+This option exports all of the patients within the specified ``[groupId]``.
+
+3.2. Exporting with filtering parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Perform a POST request to the Group endpoint, including a Parameters filter in the request body. 
+
+1. Create a POST request to the Group endpoint:
+
+        ``POST {{BASE_URL}}/Group/$export``
```

Comment:
Is it correct that you did not include a groupid in this request?

Reply:
no. fixed